src/lib/receptions/index.js

```diff
@@ -98,9 +98,9 @@ const MockReceptionTypes = {
 }
 
 const Receptions = {
-  Sequencescape: {
-    ...ReceptionTypes.Sequencescape,
-    fetchFunction: Sequencescape.fetchLabwareForReception,
+  SequencescapePlates: {
```

Comment:
Is this being restricted to just plates? Currently you can import both tubes and plates via this option. If we are restricting it to just plates we need to update the labware types for this reception.

Reply:
Thanks for the review. 
Ah, I didn't know that and just followed the story which mentioned about renaming Sequencescape to 'Sequencescape Plates'. 
 But, there is a separate option called 'Sequencescape Tubes' in the dropdown, so is it supposed to be only for plates then?